build-release.yml

```diff
@@ -0,0 +1,53 @@
+name: ReleaseBuild_$(BuildID)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+- master
+
+steps:
+
+- task: DownloadBuildArtifacts@0
+  inputs:
+    buildType: 'specific'
+    project: 'f9b79625-2860-4d92-a4ee-57b03fabfd10'
+    pipeline: '311'
+    buildVersionToDownload: 'latest'
+    downloadType: 'single'
+    downloadPath: '$(System.ArtifactsDirectory)/scripts'
+    artifactName: 'python'
+
+- checkout: self  # self represents the repo where the initial Pipelines YAML file was found
+  persistCredentials: 'true'  # set to 'true' to leave the OAuth token in the Git config after the initial fetch
+  clean: 'resources'
+  
+- task: UsePythonVersion@0
+  displayName: 'Use Python 3.x'
+
+- task: PowerShell@2
+  inputs:
+    targetType: 'inline'
+    script: |
+      . $(System.ArtifactsDirectory)/scripts/python/azure-iot-sdk-bump-version.ps1
```

Comment:
>/scripts/python/azure-iot-sdk-bump-version.ps1 [](start = 36, length = 46)

missing this file?

Reply:
This file lives in Microsoft internal repo but is called from this public yaml file.